### PR TITLE
Use updated and versioned routes

### PR DESCRIPTION
**Pull request by mjoyce91**

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -182,7 +152,7 @@ var appRouter = function (app) {
     res.status(200).send(await futureVacancies.get_future_vacancies(body$))
   });
 
-  app.get('/futureVacanciesCount', async function(req, res) {
+  app.get('/v1/futureVacanciesCount', async function(req, res) {
```
Comment (elizabeth-jimenez) [inline src/routes.js:155]: image
🐱

File: src/routes.js [hunk 2]
```diff
@@ -219,11 +189,11 @@ var appRouter = function (app) {
     res.status(200).send(await availablePositions.get_available_positions_count(body$))
   });
 
-  app.get('/positions/available/tandem', async function(req, res) {
+  app.get('/v1/cyclePositions/availableTandem', async function(req, res) {
```
Comment (elizabeth-jimenez) [inline src/routes.js:192]: ~might just be swagger docs~ Nope, NRequest matches swagger
image

File: src/routes.js (hunk 1 above)
Comment (mjoyce91) [reply src/routes.js:155]: Good find. This is a GET which we now don't use, but I will update

File: src/routes.js (hunk 2 above)
Comment (elizabeth-jimenez) [reply src/routes.js:192]: how dare you resolve me

Comment (elizabeth-jimenez) [reply src/routes.js:192]: Mike has tested UI with updated URLS, so could just be swagger docs that need to be updated

File: src/routes.js [hunk 3]
```diff
@@ -300,25 +270,25 @@ var appRouter = function (app) {
...
+  app.get('/v1/cycles', lookup(lookups.get_cycles))
+  app.get('/v1/references/grades', lookup(lookups.get_grades))
+  app.get('/v1/references/languages', lookup(lookups.get_languages))
+  app.get('/v1/posts/dangerpays', lookup(lookups.get_dangerpays))
+  app.get('/v1/posts/differentialrates', lookup(lookups.get_differentialrates))
+  app.get('/v1/posts/tourofduties', lookup(lookups.get_tourofduties))
+  app.get('/v1/fsbid/bureaus', lookup(lookups.get_bureaus))
+  app.get('/v1/references/skills', lookup(lookups.get_codes))
+  app.get('/v1/references/Locations', lookup(lookups.get_locations))
   app.get('/v1/agendaItemStatuses', lookup(lookups.get_agenda_item_statuses))
```
Comment (elizabeth-jimenez) [inline src/routes.js:283]: I know this isn't for this PR but dropping here since im checking all these routes anyways

image

File: src/routes.js [hunk 4]
```diff
@@ -349,14 +319,14 @@ var appRouter = function (app) {
   })
 
   // Need to update redundant routes once we are done with available bidders
-  app.get("/cdo/availablebidders", async function (req, res) {
+  app.get("/v1/cdo/availablebidders", async function (req, res) {
```
Comment (elizabeth-jimenez) [inline src/routes.js:322]: image
rip - these are the only AB EPs

File: src/routes.js [hunk 5]
```diff
@@ -349,14 +319,14 @@ var appRouter = function (app) {
...
     if (!req.headers.jwtauthorization) {
       res.status(200).send({ Data: null, usl_id: 4000004, return_code: -1 })
     }
     res.status(200).send(await availableBidders.get_available_bidders());
   });
 
-  app.get("/clients/availablebidders/cdo", async function (req, res) {
+  app.get("/v1/clients/availablebidders/cdo", async function (req, res) {
```
Comment (elizabeth-jimenez) [inline src/routes.js:329]: 👍

**Review by elizabeth-jimenez — CHANGES_REQUESTED** [review]
A few comments that may not even be blockers

File: src/routes.js (hunk 4 above)
Comment (mjoyce91) [reply src/routes.js]: removed deprecated route

**Pull request merged**